### ISSUE TITLE
Not able to view to react native stories on web

### ISSUE THREAD

**Issue by rajatrao777** [storybookjs/storybook#9247]
<b>Already created issue Link in design system</b>
[Link](https://github.com/storybookjs/design-system/issues/138)

**Comment by vinipachecov** [comment]
@rajatrao777  As far as I know you can't show them on the web unless you use react-native-web project.

Basically you can't render React-Native primitives on the web with storybook.
 One workaround would be to have a specific build, like with environment variables to delivery a storybook app to your testers.
There is react-native-web, which would lead to an extra effort of maintaining another project just for storybook. I think it's counter intuitive and counterproductive. 

My main idea today is to use [diawi](https://www.diawi.com/) to build a "storybook app" to show storybook components.  Fastlane can help with build automation so this can be a way to solve this.
With Diawi you can upload  your app build and they generate an installable link. With that you can share with anyone. By entering the link in the default phone browser they will be able to install the app and check storybook components.

But I would also be very happy to be wrong about this. So far this is the best way I had found.

**Comment by rajatrao777** [comment]
@vinipachecov Thanks for your response.
what you have suggested with diawi and share storybook app can be considered :+1:
Just wanted to keep it open so people who are facing the same issue/ or solved the same can share their views

**Comment by jamesjara** [comment]
Agreed, It's a bit confusing how the docs are explained, by reading the docs it seemed that you are supporting story books for reaact native. When you read "demos of react native web and react dates...etc" you can assume that those are examples of react native applications with storybook static builds. 

So yes adding a flag to build the app with the storybook registered app,  will be the only solution for now. 

thx

**Comment by shilman** [comment]
we DO support storybooks for react native -- on your device in the actual native environment, which AFAIK is what actually matters in the end. there are ways to emulate this in the browser but at the end of the day they are just emulations of varying fidelities.

**Comment by jamesjara** [comment]
yep, this is because I was understanding (and other guys in stackoverflow) that we can build a static website from the react-native storybook components, just like the title of this thread, because if you go to the docs the title "Exporting Storybook as a Static App" sounds like we can export our mobile application storybook stories to a static app(web-app) just like the linked examples, maybe adding something like `this works for react and react-native-web only, for react-native applications creating the static web application is not possible, but you can still configure your CD/CI or local environment to generate a build with the storybook application only, so the testers can work over it, etc" 

`Exporting Storybook as a Static App

But Storybook is also a tool you can use to showcase your components to others. Demos of React Native Web and React Dates are a good example for that.

For that, Storybook comes with a tool to export your storybook into a static web app. Then you can deploy it to GitHub pages or any static hosting service.`
